execution/executor/tests/db_bootstrapper_test.rs

```diff
@@ -301,22 +309,38 @@ fn test_pre_genesis() {
 #[test]
 fn test_new_genesis() {
     let (config, genesis_key) = config_builder::test_config();
-
     // Create bootstrapped DB.
     let tmp_dir = TempPath::new();
     let db = DbReaderWriter::new(LibraDB::new(&tmp_dir));
-    let genesis_txn = get_genesis_txn(&config).unwrap();
-    bootstrap_db_if_empty::<LibraVM>(&db, genesis_txn).unwrap();
+    let waypoint = {
+        let genesis_txn = get_genesis_txn(&config).unwrap();
+        bootstrap_db_if_empty::<LibraVM>(&db, genesis_txn)
+            .unwrap()
+            .unwrap()
+    };
+    let address = config.validator_network.unwrap().peer_id;
+    let signer = ValidatorSigner::new(
+        address,
+        config
+            .test
```

Comment:
maybe we should just make a function that says consensus_private_key in the test config?

Reply:
please